rosdep/python.yaml

```diff
@@ -2811,6 +2811,10 @@ python-plyfile-pip:
 python-poster:
   debian: [python-poster]
   ubuntu: [python-poster]
+python-pqdict:
+  ubuntu:
```

Comment:
Also, if we are going to make it a `-pip` key, we may as well enable it for fedora, debian, and osx as well.

Reply:
Done. Thanks for the review.